doc/source/administrator/security.md

```diff
@@ -50,6 +50,23 @@ changes to your `config.yaml` file:
 2. Apply the config changes by running `helm upgrade ...`
 3. Wait for about a minute, now your hub should be HTTPS enabled!
 
+***
+**NOTE:**
+
+If the proxy service is of type `LoadBalancer`, then a specific static IP address can be requested (if available) instead of a dynamically acquired one.  
+Although not essential for HTTPS, using a static IP address is a recommended practice for domain names referencing fixed IPs.
+This ensures the same IP address for multiple deployments.
+The IP can be provided like:
+
+```yaml
+  proxy:
+    service:
+      loadBalancerIP: xxx.xxx.xxx.xxx
+```
+
+More info about this can be found on the [Configuration Reference](https://zero-to-jupyterhub.readthedocs.io/en/latest/reference/reference.html) page.
```

Comment:
I'm not a 100%, but I think this could reference the relative path rather than a full internet url. There may be a deviation or two in these docs as there may be differences between markdown in a .md file, or rST in a rST file etc.

Could you have a look for a relative reference?

This LGTM otherwise! Thanks @GeorgianaElena !

Reply:
Thanks @consideRatio :sparkles: 
I just added a commit that uses a relative path rather than the full url of the docs section.
I also moved the docs from schema.yml about loadBalancerIp from hub to proxy. Is this ok?